packages/address/src/address.ts

```diff
@@ -53,7 +58,8 @@ export default class Address extends AbstractAddress {
 
   /**
    * Converts and returns the `bech32Address` property to a 256 bit hash string
-   *
+   * @deprecated
+   * The `bech32Address` is now deprecated. Hexadecimal will be the only address format from this point onward.
```

Comment:
```suggestion

```
We're not deprecating `B256`, so we don't need this disclaimer

Reply:
Maybe a note for these deprecations, saying to favour `toB256`?

Reply:
9d2580723